openedx/features/idea/tests/test_helpers.py

```diff
@@ -10,3 +15,57 @@ def test_upload_to_path(self):
         """Assert that file upload path is in specific format i.e. app_label/folder/filename"""
         expected_result = upload_to_path(Idea, 'my_file.jpg', 'images')
         self.assertEqual(expected_result, 'idea/images/my_file.jpg')
+
+    def test_validate_image_dimension_successfully(self):
+        """Test valid image dimension"""
+        image = Mock()
+        image.width = IDEA_IMAGE_WIDTH
+        image.height = IDEA_IMAGE_HEIGHT
+
+        try:
+            validate_image_dimension(image)
+        except ValidationError:
+            self.fail('Validation error raised, invalid dimensions')
+
+    def test_validate_image_dimension_failed(self):
+        """Test invalid image dimensions"""
+        image = Mock()
+
+        # invalid width only
+        with self.assertRaises(ValidationError):
+            image.width = 100
+            image.height = IDEA_IMAGE_HEIGHT
+            validate_image_dimension(image)
+
+        # invalid height only
+        with self.assertRaises(ValidationError):
+            image.width = IDEA_IMAGE_WIDTH
+            image.height = 100
+            validate_image_dimension(image)
+
+        # invalid width and height
+        with self.assertRaises(ValidationError):
+            image.width = 100
+            image.height = 200
+            validate_image_dimension(image)
+
+    def test_validate_file_size_successfully(self):
+        """Test valid file size"""
+        idea_file = Mock()
+        idea_file.size = 4 * 1024 * 1024
+
+        try:
+            validate_file_size(idea_file.size, max_allowed_size=4 * 1024 * 1024)
+        except ValidationError:
+            self.fail('Validation error raised, invalid file size')
```

Comment:
Let's please use `assertRaises`
https://stackoverflow.com/questions/49954928/how-to-correctly-use-assertraises-in-django
https://stackoverflow.com/a/129610

Reply:
Please avoid quoting wrong references. This test case needs opposite of `assertRaises` because it establishes that an Exception is not raised in a given circumstance [see](https://stackoverflow.com/a/4319870).

`assertRaises` has already been used in negative test case in `test_validate_file_size_failed`

Reply:
Happy to see the corrections, Actually I misunderstood the test case, I apologies for this, Thank you for making the correction @zahid-arbisoft